R/addToolbar.R

```diff
@@ -0,0 +1,111 @@
+
+
+##' @title Prepare arguments for addDrawToolbar or addPmToolbar
+##' @param fun Function used by editor package (leafpm or
+##'     leaflet.extras) to set defaults
+##' @param args Either a (possibly nested) list of named options of
+##'     the form suitable for passage to \code{fun} or (if the chosen
+##'     editor is \code{"leaflet.extras"}) \code{FALSE}.
+##' @return An object suitable for passing in as the supplied argument
+##'     to one of the formals of either for passage to either
+##'     \code{leaflet.extras::addDrawToolbar} or
+##'     \code{leafpm::addPmToolbar}.
+processOpts <- function(fun, args) {
+    ## Account for special meaning of `FALSE` as arg in leaflet.extras
+    if(isFALSE(args)) {
```

Comment:
can we use https://github.com/r-spatial/mapview/issues/177 technique instead?  I would prefer not to force users to have to upgrade to newer version of `R`.  I am sensitive to this since this has blocked me inside restrictive employers from using some packages.

Reply:
I added in `develop` branch https://github.com/r-spatial/mapedit/commit/6b206f8ddec62ca8b1bd5fdb12f5c0a8c8925d40#diff-ad5b576355b0f070a49b21f08b1c27ffR15.  @tim-salabim, please let me know if you encountered any challenges with this replacement in `mapview`.  Thanks.

Reply:
actually @tim-salabim @JoshOBrien this should be only match `FALSE` exactly so I changed in https://github.com/r-spatial/mapedit/commit/bb5e7c90db9a2550be9c731d5b70edb48b5f927c#diff-ad5b576355b0f070a49b21f08b1c27ffR14.  sorry.